newm-server/src/main/kotlin/io/newm/server/features/song/model/AudioStreamResponse.kt

```diff
@@ -4,10 +4,9 @@ import kotlinx.serialization.Serializable
 
 @Serializable
 data class AudioStreamResponse(
-    val url: String,
-    val cookies: Map<String, String>
+    val url: String
```

Comment:
Removed cookie argument because it is not required in the response (cookies are passed in the header)